Consider property group names when searching

wesionaire opened the pull request:
When searching for property groups, the name of the property group was ignored before.


### 1. Why is this change necessary?
The property search does not consider the name at all. If you search for a property by name you get zero results if this group does not have a property option with the query term contained.

### 2. What does this change do, exactly?
Adds the name of the property group to the search ranking.

### 3. Describe each step to reproduce the issue or behavior.
Create a new property with no options. Try to find it using the administration's search. It does not work.

Fresh installed shop with demo data installed **without ElasticSearch**:
![image](https://user-images.githubusercontent.com/45566296/97362665-a27f8700-18a1-11eb-8b4a-0b431a8e1c7b.png)
![image](https://user-images.githubusercontent.com/45566296/97362790-cfcc3500-18a1-11eb-9dfa-493205cc86b7.png)


### 4. Please link to the relevant issues (if any).
https://issues.shopware.com/issues/NEXT-8967

### 5. Checklist

- [ ] I have written tests and verified that they fail without my change
- [X] I have squashed any insignificant commits
- [] I have created a [changelog file](https://github.com/shopware/platform/blob/master/adr/2020-08-03-Implement-New-Changelog.md) with all necessary information about my changes
- [ ] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [X] I have read the contribution requirements and fulfil them.

mitelg commented on engine/Shopware/Controllers/Frontend/AjaxSearch.php:
```diff
@@ -60,6 +60,25 @@ public function indexAction()
         $result = $this->get('shopware_search.product_search')->search($criteria, $context);
 
+        //check if search-tearm is a valid product-number
+        if ((Shopware()->Config()->get('activateNumberSearch') == "1")) {
```
try to avoid the using of the Shopwar() singleton.
as you are in a controller, you can use `$this->container->get('')` or `$this->get('')` instead

eg: `$this->get('config')->get()`

mnaczenski replied:
thank you for your hint, i changed my PR according to your recommendation :+1:

wesionaire committed:
Consider property group names when searching

When searching for property groups, the name of the property group was ignored before.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/shopware/platform?pullRequest=1463) <br/>All committers have signed the CLA.

wesionaire commented:
@shyim can you help us here? This is a one liner and our customers need this patch for our plugin. Could you please merge it? You would help us a lot! :wink:

J-Rahe commented:
Hey @wesionaire, thanks for your contribution! You patch should be in the next release. If your customer needs this patch before that you will have to add it to his System manually.
